content/en/docs/getting_started/api-reference.md

```diff
@@ -1288,39 +1315,54 @@ spec:
   scopes:
   - PackageManifestScope
 test:
+  kubeconform:
+    kubernetesVersion: elitr
+    schemaLocations:
+    - sed
   template:
   - context:
       config: runtime.RawExtension
       environment:
+        hyperShift:
+          hostedCluster:
+            hostedClusterNamespace: sadipscing
+            metadata:
+              annotations: map[string]string
+              labels: map[string]string
+              name: amet
+              namespace: consetetur
         kubernetes:
-          version: elitr
+          version: tempor
         openShift:
-          version: sed
+          managed:
+            data: map[string]string
+          version: lorem
         proxy:
-          httpProxy: diam
-          httpsProxy: nonumy
-          noProxy: eirmod
+          httpProxy: ipsum
+          httpsProxy: dolor
+          noProxy: sit
       package:
+        image: eirmod
         metadata:
           annotations: map[string]string
           labels: map[string]string
-          name: consetetur
-          namespace: sadipscing
-    name: amet
+          name: diam
+          namespace: nonumy
+    name: sed
 
 ```
 
 
 | Field | Description |
 | ----- | ----------- |
 | `metadata` <br>metav1.ObjectMeta |  |
-| `spec` <br><a href="#packagemanifestspec">PackageManifestSpec</a> | PackageManifestSpec represents the spec of the packagemanifest containing the details about phases and availability probes. |
+| `spec` <br><a href="#packagemanifestspec">PackageManifestSpec</a> | PackageManifestSpec represents the spec of the packagemanifest containing the<br>details about phases and availability probes. |
 | `test` <br><a href="#packagemanifesttest">PackageManifestTest</a> | PackageManifestTest configures test cases. |
 
 
 ### PackageManifestLock
 
-
+PackageManifestLock allows locking packages to specific versions.
```

Comment:
It might be useful to give a version lock example here.

Reply:
An auto-generated example begins just a few lines below :+1: